### PR TITLE
added success output when new settings update

### PR DESCRIPTION
**Pull request by lestraza**

### CODE REVIEW

File: client/src/components/Mainchatpage/SettingsBar/ChangeClientData.tsx [hunk 1]
```diff
@@ -37,76 +40,83 @@ export default class SettingsClientData extends React.Component<
...
+        const { authStoreServerSuccess } = this.authStore
         return (
-            <form
-                className="client-settings__data-change-form form"
-                onSubmit={this.onSubmitForm}
-            >
-                <div className="form-row">
-                    <div className="client-settings__data-change-title form-label">
-                        Name
+            <>
```
Comment (stahlhammer88) [inline client/src/components/Mainchatpage/SettingsBar/ChangeClientData.tsx:45]: empty braces not needed

File: client/src/components/Mainchatpage/SettingsBar/ChangeClientData.tsx [hunk 2]
```diff
@@ -37,76 +40,83 @@ export default class SettingsClientData extends React.Component<
...
                     />
-                </div>
-                <input
-                    type="submit"
-                    value="Save"
-                    className={'button button--primary button--small'}
-                />
-            </form>
+                    {authStoreServerSuccess && (
+                        <div className="message">{authStoreServerSuccess}</div>
```
Comment (stahlhammer88) [inline client/src/components/Mainchatpage/SettingsBar/ChangeClientData.tsx:116]: Create a new component similar to Error.tsx

File: client/src/components/Mainchatpage/SettingsBar/ChangeClientData.tsx (hunk 1 above)
Comment (stahlhammer88) [inline client/src/components/Mainchatpage/SettingsBar/ChangeClientData.tsx:45]: Empty braces not needed

Comment (lestraza) [reply client/src/components/Mainchatpage/SettingsBar/ChangeClientData.tsx:45]: deleted it

File: client/src/components/Auth/store/Auth.store.ts [hunk 3]
```diff
@@ -47,10 +47,10 @@ export class AuthStore extends AbstractStore {
     public isAuthenticated: boolean = false
 
     @observable
-    public isRegistered: boolean = false
+    public clientId: string = ''
 
     @observable
-    public clientId: string = ''
+    public serverError: string = ''
```
Comment (stahlhammer88) [inline client/src/components/Auth/store/Auth.store.ts:53]: remove serverError

**Pull request merged**